tests/Features/ReturnTypes/BuilderExtension.php

```diff
@@ -166,6 +166,12 @@ public function testFromQuery(): Collection
     {
         return User::fromQuery('SELECT * FROM users');
     }
+
+    public function testOrWhereWithQueryExpression(): Builder
```

Comment:
I think the test in the `tests/Features/Methods/ModelExtension.php` file is enough. We can remove this.

Reply:
I'll make sure to update tomorrow or during my weekend!